smooth/components/component_battery.py

```diff
@@ -202,24 +208,19 @@ def prepare_simulation(self, components):
 
         # Max. chargeable or dischargeable power [W] going in from the bus
         # due to c_rate depending on the soc. To ensure that the battery can
-        # be fully charged in one timestep, the nominal value of the input-flow
-        # needs to be higher than what's actually going into the battery.
+        # be charged with the power indicated by the c-rate
+        # (p_charge_max = battery_capacity * c_rate_charge), the nominal value of the input-flow
+        # needs to be higher than what's actually going into the battery
+        # (p_in_max = p_charge_max / efficiency_charge).
         # Therefore we need to divide by the efficiency_charge.  Due to the
         # inflow_conversion_factor (in "create oemof model") the battery will
         # then receive right amount.
         # Flows are calculated as Power [W], so the amount of charge- or dischargeable
         # energy [Wh] has to be divided by the intervaltime [h]  (P = E / t)
         # The c-rate is given in [W/Wh], multiplication with capacity [Wh] results in power [W]
 
-        self.p_in_max = min(
-            self.c_rate_charge * self.battery_capacity,
-            (self.battery_capacity - self.soc * self.battery_capacity) /
-            (self.sim_params.interval_time/60)
-            ) / self.efficiency_charge
-        self.p_out_max = min(
-            self.c_rate_discharge * self.battery_capacity,
-            (self.soc * self.battery_capacity) / (self.sim_params.interval_time/60)
-            )
+        self.p_in_max = self.c_rate_charge * self.battery_capacity / self.efficiency_charge
+        self.p_out_max = self.c_rate_discharge * self.battery_capacity
```

Comment:
I do not understand why losses should only be considered in one direction. Either the (dis-) charging limitations lie within the battery or the limitations define the possible power withdrawl/feed-in of the battery

Reply:
okay, I can see that everything else will lead to a lot of confusion. 
With 
`        self.p_in_max = self.c_rate_charge * self.battery_capacity / self.efficiency_charge`
`        self.p_out_max = self.c_rate_discharge * self.battery_capacity * self.efficiency_discharge`
it is reads as "c_rate_carge = c_rate_discharge = 1" means you can fully charge and discharge a battery within one hour.
And when the loss rate is set to 0, charging and discharging are symmetrical.
![grafik](https://user-images.githubusercontent.com/39203376/104440360-8a5f0000-5592-11eb-9a2e-9ca5e1b944d8.png)
I agree that makes the most sense.